src/main/java/ui/LoginDialog.java

```diff
@@ -219,6 +225,9 @@ private void populateSavedFields() {
             passwordField.setText(lastLoginPassword);
         }
 
+        loadCredentialsFromFile();
+        resolveCredentials();
```

Comment:
Since `loadCredentialsFromFile` already calls `resolveCredentials`, is there a need to call `resolveCredentials` here?